quickfixj-core/src/main/java/quickfix/SessionState.java

```diff
@@ -20,10 +20,7 @@
 package quickfix;
 
 import java.io.IOException;
-import java.util.Collection;
-import java.util.Date;
-import java.util.LinkedHashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Wildcard import... you know... ;)

Reply:
Sorry, did not manage to find some code style tool which checks this automatically.